applications/nvidia_nim/nvidia_nim_chat/requirements.txt

```
@@ -0,0 +1,3 @@
+halo==0.0.31
+holoscan==1.0.3
```

Nitpick: could we pin to the latest tested version (v2.1.0)?
```suggestion
holoscan==2.1.0
```